attributes/default.rb

```diff
@@ -1 +1,2 @@
 default['gauge']['version'] = 'latest'
+default['platform_family'] = 'centos'
```

Comment:
This value is provided by chef, please remove. See https://docs.chef.io/attributes.html#automatic-ohai for other attributes provided by chef.